services/cirrus/cirrus.service.js

```diff
@@ -0,0 +1,76 @@
+'use strict'
+
+const Joi = require('joi')
+const { isBuildStatus, renderBuildStatusBadge } = require('../build-status')
+const { BaseSvgScrapingService } = require('..')
+
+const schema = Joi.object({
+  message: Joi.alternatives()
+    .try(isBuildStatus, Joi.equal('unknown'))
+    .required(),
+}).required()
+
+module.exports = class Cirrus extends BaseSvgScrapingService {
+  static get category() {
+    return 'build'
+  }
+
+  String route(String reponame) {
```

Comment:
Should it be:

```suggestion
  static get route(String reponame) {
```

?

Reply:
You may want to reference our [Tutorial](https://github.com/badges/shields/blob/master/doc/TUTORIAL.md#42-our-first-badge) on the route section (5) in particular

Reply:
Ok Ill take a look